openstack/resource_openstack_networking_subnet_v2.go

```diff
@@ -533,3 +609,43 @@ func waitForSubnetDelete(networkingClient *gophercloud.ServiceClient, subnetId s
 		return s, "ACTIVE", nil
 	}
 }
+
+func resourceSubnetV2AllocationPoolsCustomizeDiff(diff *schema.ResourceDiff) error {
+	if diff.Id() != "" && diff.HasChange("allocation_pools") {
+		o, n := diff.GetChange("allocation_pools")
+		oRaw := o.([]interface{})
+		nRaw := n.([]interface{})
+
+		samePools := true
```

Comment:
We can simplify this code.

```suggestion
		samePools := false
```